packages/common/src/hooks/chats/useChatsUserList.ts

```diff
@@ -0,0 +1,31 @@
+import { useSelector } from 'react-redux'
+
+import { Status } from 'models/Status'
+import { getUserId } from 'store/account/selectors'
+import {
+  getChats,
+  getChatsStatus,
+  getHasMoreChats
+} from 'store/pages/chat/selectors'
+import { decodeHashId } from 'utils/hashIds'
+import { removeNullable } from 'utils/typeUtils'
+
+export const useChatsUserList = () => {
+  const currentUserId = useSelector(getUserId)
+  const chats = useSelector(getChats)
+  const hasMoreChats = useSelector(getHasMoreChats)
+  const chatsStatus = useSelector(getChatsStatus)
+  const chatUserListIds = chats
```

Comment:
Any use in memoizing this and/or using the fancy selector helper we wrote to track all of these dependencies and only recompute on change? Or do we only run it once?

Reply:
Great call-out, will look into reselect for this. I don't think the other helper works here because it operates on individual chats but let me double check